cmd/frcli/csv.go

```diff
@@ -21,8 +21,8 @@ func csv(e *frdrpc.ReportEntry) string {
 
 	ts := time.Unix(int64(e.Timestamp), 0)
 
-	return fmt.Sprintf("%v,%v,%v,%v%v,%v%v,%v,%v,%v,%v,%v",
-		ts, e.OnChain, e.Type, amountPrefix, e.Amount, amountPrefix,
-		e.Fiat, e.Txid, e.Reference, e.BtcPrice.Price,
+	return fmt.Sprintf("%v,%v,%v,%v,%v%v,%v%v,%v,%v,%v,%v,%v",
```

Comment:
Missing comma after value!

Reply:
This is actually intentional, because the double `%v%v` is for an `amountPrefix` and `amount` to show whether amounts are negative. Should really use a library for this rather than rollin our own csv, on the to do list 😬

Reply:
Oops, I missed that, sorry :sweat_smile: